django/utils/timezone.py

```diff
@@ -249,3 +248,29 @@ def make_naive(value, timezone=None):
     if is_naive(value):
         raise ValueError("make_naive() cannot be applied to a naive datetime")
     return value.astimezone(timezone).replace(tzinfo=None)
+
+
+_PYTZ_BASE_CLASSES = None
```

Comment:
why not define this tuple at import time? `import pytz` seems to already have made the classes accessible.

Reply:
This was originally part of `pytz-deprecation-shim` and had all kinds of guards to avoid any unnecessary access to `pytz`. When I first migrated it to Django, it was in another file that didn't directly import `pytz`, but you're right that there's no point trying to be lazy in `django.utils.timezone`, so I've gone ahead and moved it to populating at import time.